src/telepathy/Makefile.am

```diff
@@ -46,4 +48,4 @@ telepathy_sipe_LDADD = \
 	$(DBUS_GLIB_LIBS) \
 	$(GIO_LIBS) \
 	$(GOBJECT_LIBS) \
-	$(GLIB_LIBS)
+	$(GLIB_LIBS) -lpurple
```

Comment:
I just looked at the link failures I got for telepathy and looked where the syms were
defined. I guess there is more to it and I suspect the code has to be reorganized
for telepathy to build.